src/06-loops.js

```diff
@@ -137,7 +188,15 @@ function firstIndexFound(array, target) {}
  *  lastIndexFound([ "left", "right", "left" ], "up");
  *  //> -1
  */
-function lastIndexFound(array, target) {}
+function lastIndexFound(array, target) {
+  let lastIndex =-1
+  for (let i = 0; i < array.length; i++){
+    if (array [i] === target){
+      lastIndex = i
+    }
+  }
```

Comment:
Could you write a more effecient solution by iterating backwards through the array?